script/BetterScript.s.sol

```diff
@@ -2,26 +2,38 @@
 pragma solidity ^0.8.23;
 
 import {Script, console2 as console} from "forge-std/Script.sol";
+import {Sphinx, Network} from "@sphinx-labs/plugins/SphinxPlugin.sol";
 
 interface ICREATE3Factory {
     function deploy(bytes32 salt, bytes memory creationCode) external payable returns (address deployed);
 
     function getDeployed(address deployer, bytes32 salt) external view returns (address deployed);
 }
 
-contract BetterScript is Script {
+contract BetterScript is Script, Sphinx {
     ICREATE3Factory internal factory = ICREATE3Factory(0x2Dfcc7415D89af828cbef005F0d072D8b3F23183);
 
-    modifier broadcast(uint256 key) {
-        vm.startBroadcast(key);
-        _;
-        vm.stopBroadcast();
-    }
-
-    modifier broadcastAddr(address who) {
-        vm.startBroadcast(who);
-        _;
-        vm.stopBroadcast();
+    function setUp() external {
+        sphinxConfig.owners = [address(0)]; // Add owner address(es)
+        sphinxConfig.orgId = ""; // Add org ID
+        sphinxConfig.mainnets = [
+            Network.arbitrum,
+            Network.avalanche,
+            Network.base,
+            Network.ethereum,
+            Network.linea,
+            Network.optimism,
+            Network.polygon
+        ];
+        sphinxConfig.testnets = [
+            Network.arbitrum_sepolia,
+            Network.avalanche_fuji,
+            Network.linea_goerli,
+            Network.polygon_mumbai,
+            Network.sepolia
+        ];
```

Comment:
We don't deploy on Optimism Sepolia and Base Sepolia here because the `CREATE3` factory that you use (at `0x2Dfcc7415D89af828cbef005F0d072D8b3F23183`) isn't currently deployed on these networks